src/js/media-controller.js

```diff
@@ -17,6 +17,9 @@ import { fullscreenApi } from './utils/fullscreenApi.js';
 import { constToCamel } from './utils/stringUtils.js';
 import { containsComposedNode } from './utils/element-utils.js';
 
+import MediaChromeRange from './media-chrome-range.js';
+import MediaChromeButton from './media-chrome-button.js';
```

Comment:
The `keysUsed` vs `stopPropagation` should be resolved before this PR can be merged.

Reply:
I created two PRs with examples of `keysUsed` and `stopPropagation`:
- keysUsed https://github.com/gkatsev/media-chrome/pull/1
- stopPropagation https://github.com/gkatsev/media-chrome/pull/2

To me, keysUsed is actually a bit less presumptuous since it requires less additional code from components, and it doesn't prevent users from still being able to listen to the key events if they should want.
In comparison, by default, with the stopPropagation thing, we either need to have special handling for native elements like buttons and inputs with hardcoded keys that are used, or our wrapper needs to specifically call stopPropagation, otherwise, both the native behavior and the keyboard shortcut would end up triggering. And now, we're also limiting the ability of users to be able to listen to keyup with these as targets.

Reply:
follow-up from chat:
- write some docs on how this would be used by folks
- should this be an attribute instead of a property? Or should it be both?
- better name than keysUsed?
- keysUsed doesn't preclude from media-chrome users using stopPropagation in their keyboard handling to prevent multiple behaviors from occurring.

Reply:
I'm leaning more towards stopPropagation, seems this is exactly meant for this and no extra API is added. less coupling. the little extra code in the components is worth that I think.

Reply:
Agreed on both property `keysUsed` and attribute `keyused`.